### PR TITLE
shutdown before start to avoid race conditions

### PR DESCRIPTION
**Pull request by thomas-gerber**
If Faros CE is running when one launches start, the script exits.
This ensure a consistent interaction between services on start (e.g. metabase container does not start until db is configured)

## Type of change
(Delete what does not apply)
- Bug fix (non-breaking change which fixes an issue)

# Checklist
(Delete what does not apply)
- [x] Have you checked to there aren't other open Pull Requests for the same update/change?
- [x] Have you lint your code locally before submission?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you successfully run tests with your changes locally?

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -44,6 +44,9 @@ function parseFlags() {
 }
 
 main() {
+    # For consistency when running start on running containers
+    docker compose down
```
Comment (tovbinm) [inline start.sh:48]: This seems too extreme to me. Lets error out and exit if any of our containers a running but not try to stop them. It's not a "restart" script. 

Perhaps just add "stop.sh" instead?!

Comment (thomas-gerber) [reply start.sh:48]: Why is it extreme? the outcome is the same as right now (the product is started), without any errors due to race conditions.

Comment (thomas-gerber) [reply start.sh]: I separated start and stop @tovbinm ; note that all `docker compose` commands are executed in the context of a docker-compose file (by default the one in the folder in which you execute the command)

File: stop.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```
Comment (tovbinm) [inline stop.sh:1]: please do `chmod +x on stop.sh`, commit and push the changes

**Review by tovbinm — APPROVED** [review]

**Pull request merged**